text/chapter3.md

```diff
@@ -547,11 +579,23 @@ Note that, just like for top-level declarations, it was not necessary to specify
 
 ## Infix Function Application
 
-In the code for `findEntry` above, we used a different form of function application: the `head` function was applied to the expression `filter filterEntry book` by using the infix `$` symbol.
+Most of the functions discussed so far used _prefix_ function application, where the function name was put _before_ the arguments. For example, when using the `findEntry` function to search an `AddressBook`, one might write:
```

Comment:
Let's pick a [binary function](https://github.com/purescript/documentation/blob/master/language/Syntax.md#binary-operators) (would also be good to link to that page) to compare and contrast infix vs prefix. For example `insertEntry`.

Then we can provide examples like this:
```purs
book1 = insertEntry john emptyBook
book2 = john `insertEntry` emptyBook
book3 = insertEntry john (insertEntry peggy (insertEntry ned emptyBook))
book4 = john `insertEntry` (peggy `insertEntry` (ned `insertEntry` emptyBook))

-- Shorter alias/synonym and also make it right-associative so we can drop parens.
infixr 5 insertEntry as ++
book5 = john ++ peggy ++ ned ++ emptyBook

-- Segue into how $ is another general way to drop parens
book6 = insertEntry john $ insertEntry peggy $ insertEntry ned emptyBook
```

Reply:
Another option for series of examples is:
```purs
add5 x = 5 + x
add5 x = add 5 x
add5 x = (+) 5 x
add5 x = 5 `add` x
add5   = add 5
```
Described in https://github.com/purescript-contrib/purescript-book/issues/261#issuecomment-753547758

Reply:
The above example of inserting entries into a book is almost begging to be referenced again when discussing [Semigroups and Monoids](https://book.purescript.org/chapter6.html#semigroups-and-monoids) in Chapter 6.